core/src/main/scala/org/ensime/indexer/domain.scala

```diff
@@ -188,6 +189,7 @@ final case class ArrayDescriptor(fqn: DescriptorType) extends DescriptorType {
     case a: ArrayDescriptor => a.reifier
   }
   def internalString = "[" + fqn.internalString
+  def fqnString = fqn.fqnString
```

Comment:
Not sure what you mean, fqn for descriptor doens't really mean anything (I thing) its just for convenience...